### DISCUSSION TITLE
System Properties File?

### DISCUSSION THREAD

**Discussion by amracel** [qualersoft/jmeter-gradle-plugin#151] [Q&A] [answered]
I'm getting the following error:

``
2023-03-27T11:41:45.970-0400 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] > Cannot set the value of read-only property 'systemPropertyFiles' for extension 'jmeter' of type de.qualersoft.jmeter.gradleplugin.JMeterExtension.
``

Is there something else I should be using for this field?

When I try pasting the values into ``system properties``, I can't see where they're passed on the command line.

**Comment by mathze** [comment, marked as answer]
This property is [`ConfigurableFileCollection`](https://docs.gradle.org/current/javadoc/org/gradle/api/file/ConfigurableFileCollection.html#from-java.lang.Object...-), a special class provided by gradle. You can use its `from` method to add files.
E.g.:
```
jmeter {
  systemPropertyFiles.from("path/to/myJMeterProps.properties")
}
```

**Reply by amracel** [reply]
Thanks!
